content/docs/Reference/tokens.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Tokens"
+linkTitle: "Tokens"
+date: 2020-04-03
+weight: 5
+description: >
+  Various tokens that can be used in the commands you send to Cue
```

Comment:
I recommend updating the description to:

Specify dynamic tokens in OpenCue jobs

Reply:
To be clear, here I'm just describing the value of the string to update. I'm not describing the syntax or the way you should update the code. If you looks at similar Markdown (`.md`) files, you should be able to work out how to format the description but let us know if you're stuck.

Reply:
Oh ya, sorry my bad. I didn't pay attention to that. 
Will fix this